### ISSUE TITLE
custom_detection_single_model_evaluation.py - skipping the evaluation of detection_model-ex-166--loss-0000.993.h5 because following exception occurred: ('Keyword argument not understood:', 'interpolation')

### ISSUE THREAD

**Issue by veribranchusr** [OlafenwaMoses/ImageAI#450]
Hello,
Thank you for your precious efforts for creating this awesome library.

I trained YOLOv3 model on a custom dataset of 8 classes. Used pretrained model and training went on linux with gpu. Now I try to to evaluate the model using "DetectionModelTrainer - evaluateModel" on my windows pc - CPU only.  

The code is identical with the one in the documentation which is as follows:

**
from imageai.Detection.Custom import DetectionModelTrainer

trainer = DetectionModelTrainer()
trainer.setModelTypeAsYOLOv3()
trainer.setDataDirectory(data_directory="dataset")
trainer.evaluateModel(model_path="detection_model-ex-166--loss-0000.993.h5", json_path="detection_config.json", iou_threshold=0.5, object_threshold=0.3, nms_threshold=0.5)
**

However I could not manage to generate any outputs, the only output message is:

D:\Projects\Face_recognition\virtual_environments\face36\Scripts\python.exe D:/Projects/Image/projects/card_detector/trainer/evaluate.py
Using TensorFlow backend.
Starting Model evaluation....
skipping the evaluation of detection_model-ex-166--loss-0000.993.h5 because following exception occurred: ('Keyword argument not understood:', 'interpolation')

Process finished with exit code 0

Is there some point I am missing?

**Comment by cuneytykaya** [comment]
hi @veribranchusr 
Did you find a solution for this case?

**Comment by veribranchusr** [comment]
> hi @veribranchusr
> Did you find a solution for this case?

Hi Cüneyt, hope you are well. I could not find a solution for this.

**Comment by cuneytykaya** [comment]
Hi @veribranchusr 

With this version **tensorflow-gpu==1.13.1** i successfully evaluated the model.

They have a small warning on their website.

> For the purpose of training your detection model, we advice that you have the Tensorflow-GPU v1.13.1 installed to avoid errors: 

For detail -> [link](https://imageai.readthedocs.io/en/latest/customdetection/)

**Comment by veribranchusr** [comment]
Hi @cuneytykaya, 
Mine was a lower version and I missed that point in the documentation.
Thank you for your help!
